### PR TITLE
Utilize source field for confidence and answered

### PR DESCRIPTION
**Pull request by brandon-groundlight**
There's a bug currently for counting + multiclass where confidence can be below 0.5, which throws off previous assumptions.

### CODE REVIEW

**Comment by timmarkhuff** [comment]
Looks good to me. Should we have a test for timing? For example, run `ask_ml` and make sure it completes in some reasonable time frame. I don't know what the time frame should be. Probably something between 1 second and our default patience time of 30 seconds.

**Review by tyler-romero — APPROVED** [review]
Thanks, LGTM

**Comment by brandon-groundlight** [comment]
> Should we have a test for timing?
We've considered this before, but we've decided that such tests should live in our other testing code, and not inside the SDK tests.

File: src/groundlight/internalapi.py [hunk 1]
```diff
@@ -64,21 +64,16 @@ def iq_is_confident(iq: ImageQuery, confidence_threshold: float) -> bool:
     The only subtletie here is that currently confidence of None means
     human label, which is treated as confident.
     """
-    if iq.result.confidence is None:
```
[the remark this answers is not in the document]
Comment (brandon-groundlight) [reply src/groundlight/internalapi.py:67]: A subtle change

**Pull request merged**